src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color $time, transform $time;
+    cursor: pointer;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      color: #34568b;
```

Comment:
Would be great to create variables for all the colors too in order to be able to change them in one place